src/main.c

```diff
@@ -296,8 +297,10 @@ main (int argc, char *argv[])
 
           if (rename_status == 0)
           {
-            printf ("'%s' -> '%s'\n", file.main_argv, file.dest_name);
-
+			if (verbose)  
```

Comment:
This is looking pretty good, thanks for the contribution. Only very minor edits need to made and I'll be able to merge this.

For the code to remain consistent, spaces, not tabs, need to be used here.

In geany, under the document menu, set 'indent type' to 'spaces', and 'indent width' to 2.

You can also set the default indentation in edit -> preferences -> editor -> indentation
But that may not affect documents that you've already opened.

I saw that spaces were used in purging_rmw.c and the indentation is perfect, so no need to change this statement in the other file. Just in main.c.